### DISCUSSION TITLE
Parameter identification fixing multiple parameters as same value

### DISCUSSION THREAD

**Discussion by akim2000** [Open-Systems-Pharmacology/Forum#1792] [Questions & Problems] [answered]
As I was reading the details for how PK-Sim has done its parameter identification, I came across this: 

"To improve identifiability flymph,org for liver and spleen and fJiso,org for small and large intestine were assumed to have the same value."

By that, I assume that they were able to fix the fJiso,org for small and large intestine as the same value when doing the parameter identification. How would one do this, as I have read the document on the open systems pharmacology website (https://docs.open-systems-pharmacology.org/shared-tools-and-example-workflows/parameter-identification) and still was not able to find exactly how to do so.

**Comment by tobiasK2001** [comment, marked as answer]
Hi, you can fix or identify the same parameter for different organs like this:

In the PI select the Parameter of interest in the organs you want (here it is solubility in the small intestinal compartments for example):
![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/25585248/5b0b3e34-21c7-4dd1-9163-5efcd74161ae)
than press the upper Add button:
![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/25585248/171f3633-40bd-4226-a2ce-7b02d2fcfbe5)
click the Fixed button:
![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/25585248/19e1a283-d5a2-4e43-8a91-cd85c3bd94c5)
you can now modify the parameter in all 6 sub compartment at the same time:
with the lower Add button you can also add organs to the Parameter 
![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/25585248/fa3dafd8-7163-47b4-b692-bd5ba8a99da3)
e.g. here for caecum intestinal solubility
![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/25585248/c23ecd4e-0002-47d9-a8bc-802dcd3d7c01)
hope that might help you.

Best, Tobias

**Comment by Yuri05** [comment]
Please also see the documentation for details: https://docs.open-systems-pharmacology.org/shared-tools-and-example-workflows/parameter-identification#defining-identification-parameters
